app/src/main/java/com/cornellappdev/scoop/ui/components/post/FirstPage.kt

```diff
@@ -198,8 +198,8 @@ fun TransportationSection(
                         indication = null,
                         onClick = {
                             val type = when (item) {
-                                "Student driver" -> "studentdriver"
-                                "Taxi" -> "rideshare"
+                                "Student Driver" -> "Student Driver"
+                                "Shared Taxi" -> "Shared Taxi"
```

Comment:
Again, can probably use `items` to make more scaleable. If item is in items, use it, otherwise the empty string... for example.